docs/nv2-terraform.md

```diff
@@ -60,15 +60,86 @@ Run the following commands:
     azure_resource_group_name = "notary-4d32ffd6-rg"
     signing_key_name = "example"
     ```
+6. Configure Kubectl
```

Comment:
I would recommend adding in a step between 4 and 5 which has people populate a (please create a new one in the PR) [TFVARS file](https://developer.hashicorp.com/terraform/language/values/variables#variable-definitions-tfvars-files).